### PR TITLE
Retrieve company data and display in appropriate format for Confirm Company screen

### PR DESCRIPTION
**Pull request by PhilipGlover**
Much of this is lifted from officer filing web
- Company Profile service uses an API key, whether this ought to be replaced with OAuth2 at some later point I don't know
- ConfirmCompany service displays the data
   - maybe some of this formatting should be done in the template
   - the enumerations lookup is done from files in locales, rather than the api-enumerations repo.  This is to facilitate the language, but maybe of concern for Leads to consider

Although some of this needs to be improved and tested, are we still at the stage of adding to the repo and revising it later?

IDVA3-1129

### CODE REVIEW

File: src/utils/date.ts [hunk 1]
```diff
@@ -0,0 +1,25 @@
...
+import logger from "../lib/Logger";
+
+export const toReadableFormat = (dateToConvert: string | undefined, lang = "en"): string => {
+    if (!dateToConvert) {
+        return "";
+    }
+    const jsDate = new Date(dateToConvert);
+    const dateTime = DateTime.fromJSDate(jsDate);
+    let convertedDate;
+    switch (lang) {
```
Comment (hepsimo) [inline src/utils/date.ts:11]: May be clearer as if/else

```suggestion
if (lang) {
  convertedDate = dateTime.setLocale(lang).toFormat("d MMMM yyyy");
}
else {
  convertedDate = dateTime.setLocale("en").toFormat("d MMMM yyyy");
}
```

Comment (PhilipGlover) [reply src/utils/date.ts]: changed to if / else

**Comment by ch-code-analysis** [comment]
CI: No security warnings found

**Review by hepsimo — COMMENTED** [review]
English | Cymraeg links in page header have incorrect URLs (companyNumber request param is missing) so we get 404

**Review by hepsimo — COMMENTED** [review]
Changes look good

**Review by hepsimo — APPROVED** [review]

**Review by hepsimo — APPROVED** [review]
Approving

**Comment by ch-code-analysis** [comment]
CI: No security warnings found

**Pull request merged**